pysat/_meta.py

```diff
@@ -380,7 +380,10 @@ def __contains__(self, other):
             return True
         return False
 
-    def __repr__(self, recurse=True):
+    def __repr__(self):
```

Comment:
You might want to swap \_\_str\_\_ and \_\_repr\_\_. Generally, \_\_repr\_\_ is the more verbose and unambiguous output.